examples/assets_smoke_test/assets_smoke_test/python_and_dbt_assets.py

```diff
@@ -38,17 +45,13 @@ def country_stats(country_populations: DataFrame, continent_stats: DataFrame) ->
 defs = Definitions(
     assets=load_assets_from_current_module(),
     resources={
-        "io_manager": snowflake_pandas_io_manager.configured(
-            {
-                "account": {"env": "SNOWFLAKE_ACCOUNT"},
-                "user": {"env": "SNOWFLAKE_USER"},
-                "password": {"env": "SNOWFLAKE_PASSWORD"},
-                "database": "DEV_SANDY",
-                "warehouse": "ELEMENTL",
-            }
-        ),
-        "dbt": dbt_cli_resource.configured(
-            {"project_dir": DBT_PROJECT_DIR, "profiles_dir": DBT_PROFILES_DIR}
+        "io_manager": SnowflakePandasIOManager(
+            account=EnvVar("SNOWFLAKE_ACCOUNT"),
+            user=EnvVar("SNOWFLAKE_USER"),
+            password=EnvVar("SNOWFLAKE_PASSWORD"),
+            database="DEV_SANDY",
```

Comment:
should we make these env vars too since this is an example people might try to replicate?